### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by nishi26-ai**
I have added the line of code please check and review

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -4,6 +4,10 @@
 addition= num1+num2
 print(f"The addition of {num1} + {num2} = {addition}")
 
+age= int(input("Enter your age"))
+name = input("Enter your name")
+print(f"your name is {name} and age is {age}")
+
 age= int(input("Enter your age"))
 name = input("Enter your name")
 print(f"your name is {name} and age is {age}")
```
Comment (Nishu026) [inline app.py]: you have added the code twice

**Review by Nishu026 — COMMENTED** [review]
written twice

File: app.py (hunk 1 above)
Comment (nishi26-ai) [reply app.py]: okay I will change it

**Review by Nishu026 — COMMENTED** [review]
good

**Pull request merged**